homeassistant/components/fritz/common.py

```diff
@@ -76,12 +79,16 @@ def setup(self):
         )
 
         self.fritzstatus = FritzStatus(fc=self.connection)
+        info = self.connection.call_action("DeviceInfo:1", "GetInfo")
         if self._unique_id is None:
-            self._unique_id = self.connection.call_action("DeviceInfo:1", "GetInfo")[
-                "NewSerialNumber"
-            ]
+            serial = info["NewSerialNumber"]
```

Comment:
Reformatting this will be a breaking change?

Reply:
I'll drop, is not needed anymore now that we splitted `device_info` between device_tracker and sensors.

Reply:
Reverted